node/peer.js

```diff
@@ -460,7 +409,6 @@ PreferOutgoingHandler.prototype.getQOS = function getQOS() {
     }
 };
 
-// Consulted depending on the peer state
```

Comment:
There seems to have a bug in PreferOutgoingHandler.prototype.getQOS.
self.peer.getOutConnection() should be called with parameter "true", otherwise, it may return unidentified out connection even there is an identified out connection.